scripts/pre_commit_linter.py

```diff
@@ -563,10 +568,15 @@ def _check_bad_pattern_in_file(filename, content, pattern):
     if not (any(filename.startswith(excluded_dir)
                 for excluded_dir in pattern['excluded_dirs'])
             or filename in pattern['excluded_files']):
-        if re.search(regexp, content):
-            print '%s --> %s' % (
-                filename, pattern['message'])
-            return True
+        bad_pattern_count = 0
+        with open(filename, 'r') as file_to_check:
+            for line_num, line in enumerate(file_to_check, 1):
+                if re.search(regexp, line):
+                    print 'L: %s %s --> %s' % (
```

Comment:
Perhaps L --> Line? Also, maybe format like this:

`filename --> line 590: ...`

The idea is that devs will need to open the file by its filename, before they can even use the line number.

Reply:
Got it. Thanks!